src/rest/runners.rs

```diff
@@ -14,11 +14,11 @@ use crate::model::{GitLabRunner, GitLabRunnerConfig};
     post,
     path = "/runners",
```

Comment:
```suggestion
    path = "/gitlab-runners",
```

this and other paths